Knock.net/Knock.net.csproj

```diff
@@ -28,6 +28,10 @@
     <PackageVersion>0.1.3</PackageVersion>
   </PropertyGroup>
 
+  <ItemGroup>

```

Comment:
Could you hand edit this file and remove these item groups please. I don't know why VS adds this.

Reply:
Will do!